src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts

```diff
@@ -30,12 +30,19 @@ const BASE_URL = 'user-management/users';
 export class UserListComponent implements OnInit {
   @ViewChild('userRolesTpl', { static: true })
   userRolesTpl: TemplateRef<any>;
+  @ViewChild('warningTpl', { static: true })
+  warningTpl: TemplateRef<any>;
+  @ViewChild('durationTpl', { static: true })
+  durationTpl: TemplateRef<any>;
 
   permission: Permission;
   tableActions: CdTableAction[];
   columns: CdTableColumn[];
   users: Array<any>;
+  expirationWarning1: number;
```

Comment:
It seems to me that more meaningful var names are better to ditsinguish between the 2 types of warnings:
```
expirationWarning1 -> expirationWarningAlert (level 1)
expirationWarning1 -> expirationDangerAlert (level 2)
```
Same for USER_PWD_EXPIRATION_WARNING_1 and USER_PWD_EXPIRATION_WARNING_2